src/main/java/org/jenkinsci/plugins/github/pullrequest/data/GitHubPREnv.java

```diff
@@ -0,0 +1,43 @@
+package org.jenkinsci.plugins.github.pullrequest.data;
+
+import hudson.model.BooleanParameterValue;
+import hudson.model.ParameterValue;
+import hudson.model.StringParameterValue;
+
+import static org.apache.commons.lang.StringUtils.trimToEmpty;
+
+/**
+ * @author lanwen (Merkushev Kirill)
+ */
+public enum GitHubPREnv {
```

Comment:
manage enum is easier than set of strings. And we can find and see all env vars in one place. I've already done same with my inner jenkins plugins - it looks pretty